.babelrc

```diff
@@ -12,12 +12,6 @@
     "@babel/plugin-transform-flow-strip-types",
     "@babel/plugin-transform-property-mutators",
     "@babel/plugin-proposal-object-rest-spread",
-    [
```

Comment:
@Yuvalke did you ran sanity to check this doesn't cause regression here?
https://github.com/kaltura/playkit-js-ui/pull/534

Reply:
@dan-ziv Sure, why did you add "@babel/preset-flow" and "pragmaFrag": "Fragment"? maybe we should remove it as well if it's not needed anymore

Reply:
How's your change fixing it?